CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+# 0.5.0
+This version makes this package compatible with MeiliSearch v0.25.0 and newer
+## Changes
+- Add method to responds with raw information from API `client.getRawIndex` (#124) @brunoocasali
+- Run a `pub upgrade` in dependencies (#128) @brunoocasali
+- Add support to keys (according to v0.25.0 spec) [more](https://docs.meilisearch.com/reference/api/keys.html#keys) (#121) @brunoocasali
+  - Create a new resource class `Key` to represent keys
+  - Create methods to support keys:
+    - `client.getKeys`
```

Comment:
I'm not totally sure but I think `getKeys` method should be in breaking because the return of the method has changed.

Reply:
You are both way to fast for me 😄

Reply:
Thanks for raising this @alallema!

But if you scroll down a bit, you'll see the `client.getKeys` in the breaking section too :)

Reply:
![image](https://user-images.githubusercontent.com/4116980/150186896-50894265-f65e-4172-ab03-cb120ed8aab4.png)